### PR TITLE
Update snapshots

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

File: packages/clay-checkbox/src/ClayCheckbox.js [hunk 1]
```diff
@@ -90,10 +90,20 @@ ClayCheckbox.STATE = {
 	 * Label of the input. Required for accesibility.
 	 * @instance
 	 * @memberof ClayCheckbox
-	 * @type {!string|html}
+	 * @type {?string}
```
Comment (matuzalemsteles) [inline packages/clay-checkbox/src/ClayCheckbox.js]: Should be `{?string|undefined}`?

File: packages/clay-dropdown/src/items_validator.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import {Config} from 'metal-state';
 let itemShape = {
 	active: Config.bool().value(false),
 	checked: Config.bool().value(false),
+	data: Config.object(),
```
Comment (matuzalemsteles) [inline packages/clay-dropdown/src/items_validator.js]: Hey @carloslancha, This `data` correspond to `dataAttributes`?

Comment (carloslancha) [reply packages/clay-dropdown/src/items_validator.js]: I added wrong files in the SF commit...

**Comment by matuzalemsteles** [comment]
Merged, thanks!

**Pull request merged**